fastlane/Fastfile

```diff
@@ -58,6 +58,40 @@ platform :ios do
   end
 
 
+  desc 'Run ui tests'
+  lane :uitest do
+    system('pkill', '-9', '-x', 'Simulator')
+    UI.success('Simulator app killed')
+
+    device = ENV['TEST_DEVICE_NAME'] || 'iPhone 6s'
+    UI.message("Simulator testing device set to '#{device}'")
+
+    output_directory = 'build/tests'
```

Comment:
Change to `'build/ui-tests'` so we don’t overwrite the results of the unit tests.